bundles/automation/org.eclipse.smarthome.automation.module.core/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.module.core
 Bundle-Version: 0.9.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: com.google.common.collect,
+ org.apache.commons.lang,
```

Comment:
I am actually using this because its very convenient here: `StringUtils.isNotBlank(eventOnChannel)`

In case the user enter multiple blanks, i.e. "   " into paperUi in the event field, he will notice that and then might wonder why this trigger does not fire. The function makes a `null` check obsolete too.